src/PostgREST/ApiRequest.hs

```diff
@@ -87,6 +88,8 @@ data ApiRequest = ApiRequest {
   , iCanonicalQS :: String
   -- | JSON Web Token
   , iJWT :: T.Text
+  -- | Request Cookies
+  , iCookies :: Maybe [(T.Text, T.Text)]
```

Comment:
Is there any difference between `Just []` and `Nothing` ?

It might be easier to do something like:
```haskell
iCookies = 
  case lookupHeader "Cookie" of 
    Just hdr -> parseCookiesText hdr
    Nothing -> []
```
In this case we could drop the `fromMaybe` in the `cookieClaims` definition inside the middleware.